### ISSUE TITLE
Fail to return false for missing element(s)

### ISSUE THREAD

**Issue by prashanth-sams** [krupani/web-object#1]
```
while @driver.find_element(css: "xyx").size()!=0 do
...
end
```
or
```
while self.your_element.displayed? do
...
end
```

## Error
```
no such element: Unable to locate element: {"method":"css selector","selector":"xyz"}
```

**Comment by krupani** [comment]
This will be fixed in the next release.. Thanks

**Comment by krupani** [comment]
Issue is resolved in version 0.2, can be found in following 2 places:
* https://github.com/krupani/web-object/releases/tag/0.2
* https://rubygems.org/gems/web-object/versions/0.2
   
or     
   
please update the gem in your project.

Thanks,
K

Closing the issue 👍
